sdk/src/client/api/block_builder/input_selection/mod.rs

```diff
@@ -216,9 +217,9 @@ impl InputSelection {
         self
     }
 
-    /// Sets the timestamp of an [`InputSelection`].
-    pub fn timestamp(mut self, timestamp: u32) -> Self {
-        self.timestamp = timestamp;
+    /// Sets the slot index of an [`InputSelection`].
+    pub fn slot_index(mut self, slot_index: impl Into<SlotIndex>) -> Self {
```

Comment:
general question ... should we add a `with_` prefix to those `-> Self` methods (in a separate PR)?

Reply:
Yeah was thinking the same https://github.com/iotaledger/iota-sdk/issues/1125